components/forms/Login/index.test.jsx

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react';
+import Login, { loginFormTestIds } from '.';
+
+describe('Login', () => {
+  beforeEach(() => {
+    render(<Login />);
+  });
+
+  it('renders an email label', () => {
+    const emailLabel = screen.getByTestId(loginFormTestIds.emailLabel);
+
+    expect(emailLabel).toBeVisible();
+    expect(emailLabel).toHaveTextContent('Email');
+  });
+
+  it('renders an email input field', () => {
+    const emailInput = screen.getByTestId(loginFormTestIds.emailField);
+
+    expect(emailInput).toBeVisible();
+    expect(emailInput).toHaveAttribute('type', 'email');
+  });
+
+  it('renders an password field', () => {
+    const passwordLabel = screen.getByTestId(loginFormTestIds.passwordLabel);
+
+    expect(passwordLabel).toBeVisible();
+    expect(passwordLabel).toHaveTextContent('Password');
+  });
+
+  it('renders an password input field', () => {
```

Comment:
```suggestion
  it('renders a password input field', () => {
```

Reply:
done in https://github.com/tyrro/react-survey/pull/50/commits/0534d29c922369266f9200ad61515222e6e7a9f8